paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetActivityTest.kt

```diff
@@ -136,17 +136,6 @@ internal class CustomerSheetActivityTest {
         }
     }
 
-    @Test
-    fun `When showEditMenu is true, edit menu is visible`() {
```

Comment:
Do we really want to remove this test? Are we testing this elsewhere?

Reply:
We don’t have `showEditMenu` anymore because that’s determined by `savedPaymentMethods`, so there’s no need for a test.